src/test/scala/io/iohk/ethereum/network/handshaker/EtcHandshakerSpec.scala

```diff
@@ -158,7 +158,7 @@ class EtcHandshakerSpec extends FlatSpec with Matchers  {
       override val customGenesisFileOpt: Option[String] = None
       override val daoForkBlockTotalDifficulty: BigInt = 0
       override val chainId: Byte = 0.toByte
-      override val blockReward: UInt256 = UInt256(0)
+      override val monetaryPolicyConfig: MonetaryPolicyConfig = null
```

Comment:
`???` would only work with a `def` or a `lazy val`. In my view, `nulls` are fine in tests.